packages/vike-react-query/renderer/+config.h.ts

```diff
@@ -5,6 +5,7 @@ export default {
   queryClientConfig: undefined,
   VikeReactQueryWrapper: 'import:vike-react-query/renderer/VikeReactQueryWrapper:default',
   FallbackErrorBoundary: 'import:vike-react-query/renderer/FallbackErrorBoundary:default',
+  stream: true,
```

Comment:
Since this package use react-streaming (e.g. useStream()).
Here we need to set `stream: true`, or there will be an error